### ISSUE TITLE
Training Code

### ISSUE THREAD

**Issue by angelgaspar** [QuJX/DDNet#8]
Hello Author,

I am Angel Gaspar, a Ph.D. student in Artificial Intelligence. I read your paper and found it highly relevant to my research.
Could you please share the training code and dataset (or instructions to access it)? I will properly cite and acknowledge your work in my research.

Thank you very much for your time and consideration.

Best regards,
Angel Gaspar
Ph.D. Student, University of Guadalajara
Email: angel.gaspar@alumnos.udg.mx

**Comment by QuJX** [comment]
Hi Angel,

Thanks for your interests to our work. The dataset inculdes LOL dataset, which can be obtained from: https://figshare.com/articles/dataset/_Information_zip/27192921, and 1000 outdoor images from PASCAL VOC, COCO and DETRAC dataset. Unfortunately, I lost the random picked data, so you may randomly picked by yourself, but I remember that I used the labels of orginal data, like 'Car', to filter the dataset and keeping the transportation-related data.

About the training code, I also lost it along with the whole training dataset. Because I have graduated, and this work was done 2 years ago, it's diffucult for me to find them back. Sorry for any inconvenience.

Hope my above information could help you. 

Best Wishes
Tom JX Qu.
